core/src/main/java/com/yahoo/ycsb/workloads/RestWorkload.java

```diff
@@ -0,0 +1,300 @@
+      updateUrlMap = getTrace(p.getProperty(UPDATE_TRACE_FILE, UPDATE_TRACE_FILE_DEFAULT), updateRecordCount);
+    
+    operationchooser = createOperationGenerator(p);
+
+    // Common distribution for all operations.
+    String requestDistrib = p.getProperty(REQUEST_DISTRIBUTION_PROPERTY, REQUEST_DISTRIBUTION_PROPERTY_DEFAULT);
+
+    double readZipfconstant = Double
+        .parseDouble(p.getProperty(READ_ZIPFIAN_CONSTANT, READ_ZIPFIAN_CONSTANT_DEAFULT));
+    readKeyChooser = getKeyChooser(requestDistrib, readUrlMap.size(), readZipfconstant, p);
+    double updateZipfconstant = Double
+        .parseDouble(p.getProperty(UPDATE_ZIPFIAN_CONSTANT, UPDATE_ZIPFIAN_CONSTANT_DEAFULT));
+    updateKeyChooser = getKeyChooser(requestDistrib, updateUrlMap.size(), updateZipfconstant, p);
+    double insertZipfconstant = Double
+        .parseDouble(p.getProperty(INSERT_ZIPFIAN_CONSTANT, INSERT_ZIPFIAN_CONSTANT_DEAFULT));
+    insertKeyChooser = getKeyChooser(requestDistrib, insertUrlMap.size(), insertZipfconstant, p);
+    double deleteZipfconstant = Double
+        .parseDouble(p.getProperty(DELETE_ZIPFIAN_CONSTANT, DELETE_ZIPFIAN_CONSTANT_DEAFULT));
+    deleteKeyChooser = getKeyChooser(requestDistrib, deleteUrlMap.size(), deleteZipfconstant, p);
+
+    fieldlengthgenerator = getFieldLengthGenerator(p);
+  }
+  
+  public static DiscreteGenerator createOperationGenerator(final Properties p) {
+     // Re-using CoreWorkload method. 
+    final DiscreteGenerator operationChooser = CoreWorkload.createOperationGenerator(p);
+    // Needs special handling for delete operations not supported in CoreWorkload.
+    double deleteproportion = Double
+		        .parseDouble(p.getProperty(DELETE_PROPORTION_PROPERTY, DELETE_PROPORTION_PROPERTY_DEFAULT));
+    if (deleteproportion > 0)
+      operationChooser.addValue(deleteproportion, "DELETE");
+    return operationChooser;
+  }
+
+  private static NumberGenerator getKeyChooser(String requestDistrib, int recordCount, double zipfContant,
+      Properties p) throws WorkloadException {
+    NumberGenerator keychooser = null;
+    if (requestDistrib.compareTo("exponential") == 0) {
+      double percentile = Double.parseDouble(p.getProperty(ExponentialGenerator.EXPONENTIAL_PERCENTILE_PROPERTY,
+          ExponentialGenerator.EXPONENTIAL_PERCENTILE_DEFAULT));
+      double frac = Double.parseDouble(p.getProperty(ExponentialGenerator.EXPONENTIAL_FRAC_PROPERTY,
+          ExponentialGenerator.EXPONENTIAL_FRAC_DEFAULT));
+      keychooser = new ExponentialGenerator(percentile, recordCount * frac);
+    } else if (requestDistrib.compareTo("uniform") == 0) {
+      keychooser = new UniformIntegerGenerator(0, recordCount - 1);
+    } else if (requestDistrib.compareTo("zipfian") == 0) {
+      keychooser = new ZipfianGenerator(recordCount, zipfContant);
+    } else if (requestDistrib.compareTo("latest") == 0) {
+      // TODO: To be implemented.
+      System.err.println("Latest request distribution is not supported");

```

Comment:
Handled - `throw new WorkloadException("Latest request distribution is not supported for RestWorkload.");`